api_guard/dist/types/index.d.ts

```diff
@@ -173,7 +174,7 @@ export declare function forkJoin<A extends readonly unknown[], R>(...sourcesAndR
 export declare function forkJoin(sourcesObject: {
     [K in any]: never;
 }): Observable<never>;
-export declare function forkJoin<T>(sourcesObject: T): Observable<{
+export declare function forkJoin<T extends Record<any, ObservableInput<any>>>(sourcesObject: T): Observable<{
```

Comment:
Can this really be `Record<any`? I feel like the internal implementation will only work with strings.